config/rbac/role.yaml

```diff
@@ -7,69 +7,151 @@ metadata:
   name: manager-role
```

Comment:
In this file, there are a few rules that have the same `apiGroups` and  `verbs`.
Could you please consolidate them into a single rule rather than have multiple different ones?

Reply:
I just split the rules by different `apiGroups` and `verbs`. I think it might be more clear.

Reply:
Alright, fair enough